src/core.js

```diff
@@ -1680,11 +1680,17 @@ define([
                 isCopyable: !multiselect && mug.options.isCopyable
             }));
         $baseToolbar.find('.fd-button-remove').click(function () {
-            var mugs = _this.getCurrentlySelectedMug(true);
-            if (mugs.length > 1 || (mugs.length && form.getChildren(mugs[0]).length)) {
+            var mugs = _this.getCurrentlySelectedMug(true),
+                errorMessage;
+            if (mugs.length > 1) {
+                errorMessage = "You are about to delete multiple questions. This cannot be undone";
+            } else if (mugs.length && form.getChildren(mugs[0]).length) {
+                errorMessage = "You are about to delete a question with child nodes. This cannot be undone";
```

Comment:
Don't want to let you hanging @emord.

My vote is for one short message for both cases: "Deleting multiple questions cannot be undone."

But I'm willing to compromise. @orangejenny I'm not quite sure what your preference is. You said "shorter & more direct is better" and also want two separate messages. Do these seem good to you?

- `mugs.length > 1` -> "Deleting multiple questions cannot be undone."
- mug has children -> "Deleting a question with children cannot be undone."